### PR TITLE
[desktop][qt] 60 fps update freq.

### PR DESCRIPTION
**Pull request by vng**
Thanks to @AndrewShkrob 

@Ferenc- @pastk Please, check how it works on your Linux.

### CODE REVIEW

**Comment by Ferenc-** [comment]
I have tested this on desktop and mobile as well. So far I haven't seen any issues.

**Review by Ferenc- — APPROVED** [review]

**Comment by vng** [comment]
Did you notice any visual enhancement in rendering?

File: qt/qt_common/map_widget.cpp [hunk 1]
```diff
@@ -39,11 +39,11 @@ MapWidget::MapWidget(Framework & framework, bool isScreenshotMode, QWidget * par
   , m_contextFactory(nullptr)
 {
   setMouseTracking(true);
-  // Update widget contents each 30ms.
+  // Update widget's content frequency is 60 fps.
   m_updateTimer = std::make_unique<QTimer>(this);
   VERIFY(connect(m_updateTimer.get(), SIGNAL(timeout()), this, SLOT(update())), ());
   m_updateTimer->setSingleShot(false);
-  m_updateTimer->start(30);
+  m_updateTimer->start(16);
```
Comment (biodranik) [inline qt/qt_common/map_widget.cpp:46]: What does 16 mean?

File: qt/qt_common/map_widget.cpp [hunk 2]
```diff
@@ -39,11 +39,11 @@ MapWidget::MapWidget(Framework & framework, bool isScreenshotMode, QWidget * par
   , m_contextFactory(nullptr)
 {
   setMouseTracking(true);
-  // Update widget contents each 30ms.
+  // Update widget's content frequency is 60 fps.
```
Comment (biodranik) [inline qt/qt_common/map_widget.cpp:42]: Can we have some automatic default taken from the system for example? What's the need to force 60? Won't it consume more battery on laptops and mobile Linux devices?

**Review by biodranik — COMMENTED** [review]
@Ferenc- are there any system settings related to:
- FPS
- Power Saving mode

That we can use for some better defaults?

File: qt/qt_common/map_widget.cpp (hunk 1 above)
Comment (vng) [reply qt/qt_common/map_widget.cpp:46]: period in ms, 1000/16 ~ 60 fps

File: qt/qt_common/map_widget.cpp (hunk 2 above)
Comment (vng) [reply qt/qt_common/map_widget.cpp:42]: Well, it is not intended to consume more battery, here is how it works:
We have the drape core which calculates frames by signals (gestures, etc) as fast as possible, and even in Debug it is usually > 60 fps. But for some reason, we do check if we have an updated frame with 30 fps freq only, so we skip intermediate already calculated frames.

60 fps is an industry standard, what kind of system settings can we check here?

**Comment by Ferenc-** [comment]
> * FPS

I can't think of any such system level "setting" at the moment. I think the proposed 60 FPS is, as said, a reasonable industry standard, which doesn't try to produce a wastefully high number of frames. It's congruent with typical display refresh rates.

> * Power Saving mode

There is no single "power saving mode", but there are a whole lot of power saving related settings on the kernel level, and the different "system setting" tools are typically just exposing those.
Even the number of options are very much hardware/kernel dependent.
Most common is for example the `powersave`, `conservative`, ... `performance` [governor settings]( https://www.kernel.org/doc/html/v6.1/admin-guide/pm/cpufreq.html#generic-scaling-governors), where one would expect to have something else then `performance` in a so called `"Power Saving mode"`.

File: qt/qt_common/map_widget.cpp (hunk 1 above)
Comment (biodranik) [reply qt/qt_common/map_widget.cpp:46]:
```suggestion
  std::chrono::milliseconds constexpr kTimerPeriod {1000 / 60};
  m_updateTimer->start(kTimerPeriod);
```

**Review by biodranik — APPROVED** [review]

**Review by biodranik — APPROVED** [review]

**Comment by pastk** [comment]
> @Ferenc- @pastk Please, check how it works on your Linux.

Sorry for the delay. It works fine on my laptop!

**Pull request merged**